cas/cas.c

```diff
@@ -26,6 +26,7 @@
 #include "lib/finject.h"     /* M0_FI_ENABLED */
 #include "fid/fid.h"         /* m0_fid_type_register */
 #include "fop/fop.h"
+#include "fop/wire_xc.h"
```

Comment:
This is not needed, because it is already included in "cas/cas.h"

Reply:
This is fine. Our coding-style convention is that #include commands should not depend on the internals of .h files (otherwise they will have to be changed often).